src/components/Badge/index.d.ts

```diff
@@ -6,6 +6,7 @@ export interface BadgeProps extends BaseProps {
     title?: string;
     children?: ReactNode;
     size?: 'small' | 'medium' | 'large';
+    borderRadius?: string;
```

Comment:
@adrianestevez1 here we need to be specific

```suggestion
    borderRadius?: 'square' | 'semi-rounded' | 'rounded';
```